samples/bluetooth/mesh/light_switch/README.rst

```diff
@@ -51,36 +85,39 @@ If the model is configured to publish to a unicast address, the model handler ca
 The response from the target device updates the corresponding LED on the mesh light switch device.
 If the model is configured to publish to a group address, it calls :c:func:`bt_mesh_onoff_cli_set_unack` instead, to avoid getting responses from multiple devices at once.
 
-Requirements
-************
-
-The sample supports the following development kits:
-
-.. table-from-rows:: /includes/sample_board_rows.txt
-   :header: heading
-   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuappns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833
-
-The sample requires a smartphone with Nordic Semiconductor's nRF Mesh mobile app installed in one of the following versions:
-
-  * `nRF Mesh mobile app for Android`_
-  * `nRF Mesh mobile app for iOS`_
-
-An additional requirement is the :ref:`bluetooth_mesh_light` sample application programmed on a separate device, and configured according to the mesh light sample's :ref:`testing guide <bluetooth_mesh_light_testing>`.
+.. _bluetooth_mesh_light_switch_user_interface:
 
 User interface
 **************
 
 Buttons:
-   Buttons are used to control the respective Generic OnOff Clients.
-   When pressed, the button toggles the LED state on a :ref:`mesh light <bluetooth_mesh_light>` device.
+      During the provisioning process, the buttons (1 to 4) can be used for OOB input.
```

Comment:
Is this correct? It looks the the button handler returns before it is provisioned.
```suggestion
      During the provisioning process, the buttons (1 to 4) is not used.
```

Reply:
Here I just had a sentence from Trond to lean on (når provisioning foregår, kan knappene brukes til "OOB input"). @trond-snekvik can you please confirm that it is ok for you too if I change to Ingar's suggestion :)

Reply:
@mia-ko's text is correct, provisioning has its own button handler here:
https://github.com/nrfconnect/sdk-nrf/blob/master/subsys/bluetooth/mesh/dk_prov.c#L75-L87